clients/go-ethereum/mapper.jq

```diff
@@ -76,6 +76,7 @@ def to_bool:
         "baseFeeUpdateFraction": (if env.HIVE_OSAKA_BLOB_BASE_FEE_UPDATE_FRACTION then env.HIVE_OSAKA_BLOB_BASE_FEE_UPDATE_FRACTION|to_int else 5007716 end)
```

Comment:
We should add `"bpo1"` here, very similar to how we have `"osaka"` and `"prague"`, etc, for this and the rest of the clients.

Reply:
Interesting, I'm not sure if we should set up a default, could you check what happens if we don't put a default?
Ideally, we should only have the `"bpo1"` entry if we pass any of the `HIVE_BPO1_BLOB_...` constants IMO.

If it adds too much complexity I guess it would not hurt to add a default.